velox/type/tests/TimestampTest.cpp

```diff
@@ -418,5 +418,71 @@ TEST(TimestampTest, tmToStringTimestamp) {
   // %FT%T - equivalent to "%Y-%m-%dT%H:%M:%S" (the ISO 8601 timestamp format)
   testTmToString("%FT%T", TimestampToStringOptions::Mode::kFull);
 }
+
+TEST(TimestampTest, leadingPositiveSign) {
+  TimestampToStringOptions options = {
+      .leadingPositiveSign = true,
+      .zeroPaddingYear = true,
+      .dateTimeSeparator = ' ',
+  };
+  auto testTmToString = [&](Timestamp ts, const std::string& expected) {
+    std::tm tm;
+    Timestamp::epochToUtc(ts.getSeconds(), tm);
+    ASSERT_EQ(Timestamp::tmToString(tm, ts.getNanos(), options), expected);
```

Comment:
It is better to put assertions into the test body. Otherwise, the error message will not be able to indicate which of the calls actually failed.

ASSERT_EQ(testTmToString(...), expected);